src/main/java/profiler/generation/R1CSConstruction.java

```diff
@@ -18,16 +18,13 @@
 import scala.Tuple3;
 
 import java.io.Serializable;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
+import java.util.*;
 import java.util.Iterator;
```

Comment:
java.util.Iterator; not needed, if you import util.*, But maybe it would be better practice to not importing util.* and only import the packages, which you need.

Reply:
Yes, I didn't notice this, but assume the util.* was already there. I will get rid of it and see whats missing.

Reply:
It turns out, I didn't need to edit this file anyway. It only included "serialSmallestExample" which doesn't need to be here and a few style fixes that bothered me. This, previously existing, file now remains untouched.